Services/Data/default.json

```diff
@@ -69,6 +69,8 @@
       "Description": "Pressure > 250",
```

Comment:
Use the [default.json](https://github.com/Azure/device-telemetry-dotnet/blob/master/Services/Data/Rules/default.json) from device-telemetry-dotnet.